Makefile

```diff
@@ -55,3 +55,15 @@ CONTROLLER_TOOLS_VERSION ?= v0.9.2
 controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessary.
 $(CONTROLLER_GEN): $(LOCALBIN)
 	test -s $(LOCALBIN)/controller-gen || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_TOOLS_VERSION)
+
```

Comment:
L58 to L69 can be replaced with contents from https://github.com/nephio-project/nephio/blob/main/Makefile